lib/paged_vertical_calendar.dart

```diff
@@ -89,8 +90,11 @@ class PagedVerticalCalendar extends StatefulWidget {
   /// how many months should be loaded outside of the view. defaults to `1`
   final int invisibleMonthsThreshold;
 
-  /// list padding, defaults to `EdgeInsets.zero`
-  final EdgeInsetsGeometry listPadding;
+  /// calendar bottom padding, defaults to `0`
+  final double calendarBottomPadding;
```

Comment:
padding should always use a single EdgeInsets Object, you can use it's top and bottom parameters to build your padding around both calender lists. 

the name is a little bit confusing, maybe keep it as listpadding